Tests/MapperTests/ConvertibleValueTests.swift

```diff
@@ -128,6 +128,30 @@ final class ConvertibleValueTests: XCTestCase {
         XCTAssertNil(test.URL)
     }
 
+    func testConvertibleArrayOfKeysDoesNotThrow() {
+        struct Test: Mappable {
+            let string: String
+            init(map: Mapper) throws {
+                try self.string = map.from(["a", "b"])
+            }
+        }
+
+        let test = try? Test(map: Mapper(JSON: ["b": "someValue"]))
+        XCTAssertNotNil(test)
```

Comment:
Can you test instead that `test.string == "someValue"`?

Reply:
yea, done. Also did it for the other test case